app/components/option_item/index.tsx

```diff
@@ -125,7 +126,6 @@ const OptionItem = ({
     description,
     destructive,
     icon,
-    iconColor,
```

Comment:
If we are no longer using this, then it should be removed from the type definition. But why are we removing it?

Reply:
Thanks for pointing this out. I've added it back by using this in `option_icon.tsx`